GitSearch/Networking/HTTP/Result.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Result.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/27/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import Foundation
+
+enum Result<T> {
```

Comment:
Swift 5 already contains the very same type. Please update swift version to 5 (instead of current 4.2), and use `Result` type from library instead.

Reply:
I will, and get back to this